README.md

```diff
@@ -55,13 +56,19 @@
 - [Vesting](https://aiken-lang.org/example--vesting) - Learn how to work with time
 - [Gift Card](https://aiken-lang.org/example--gift-card) - Make gift cards using NFTs using Aiken and Deno fresh
 - [Gift Card NextJS](https://github.com/adalicious/aiken-gift-card) - A similar gift card example but with NextJS
+- [Gift Card MeshJS](https://meshjs.dev/smart-contracts/giftcard) - Gift card contract with full integration with MeshJS, offering code example and explanation.
+- [Escrow MeshJS](https://meshjs.dev/smart-contracts/escrow) - Escrow contract with full integration with MeshJS, offering code example and explanation.
+- [Marketplace MeshJS](https://meshjs.dev/smart-contracts/marketplace) - Cardano native token marketplace contract with full integration with MeshJS, offering code example and explanation.
+- [Payment Splitter MeshJS](https://meshjs.dev/smart-contracts/payment-splitter) - Payment splitter contract with full integration with MeshJS, offering code example and explanation.
+- [Simple Swap MeshJS](https://meshjs.dev/smart-contracts/swap) - Simple swap contract for token exchange with full integration with MeshJS, offering code example and explanation.
+- [Vesting Contract MeshJS](https://meshjs.dev/smart-contracts/vesting) - Vesting contract for token exchange with full integration with MeshJS, offering code example and explanation.
```

Comment:
Thanks for the submission here, but I don't think that the awesome list is the best place for those. Reason being that none of these tutorials actually contain Aiken code. Rather, they focus on the off-chain part. It seems like they would be much more appropriate on the website (https://github.com/aiken-lang/site) as "off-chain" tutorials, presenting alternative to Lucid.

Reply:
have replaced link to aiken workspace instead, as these collection of scripts has been useful for many